prime-router/docs/design/design/mapping-schemas.md

```diff
@@ -0,0 +1,77 @@
+# Mapping schema file structure
+
+Translation between HL7v2 and FHIR requires schema files that define how data is translated from one format to another. As we have built out these mappings, shortcomings in the current layout of the files have become apparent. This proposed structure aims to be self-documenting and logical so that the mappings are more maintainable and reusable.
```

Comment:
It's a nit, but I think the HL7->FHIR files are called [templates](https://github.com/LinuxForHealth/hl7v2-fhir-converter/blob/master/TEMPLATING.md). But I guess we can refer to them as schemas for our purposes.

Reply:
Arnej suggested on an earlier draft it would be less confusing if we used schema across the board.